packages/dashboard/src/components/confirmModal/confirmModal.css

```diff
@@ -0,0 +1,9 @@
+.btn-custom-primary {
+  border-color: var(--colors-background-button) !important;
+  background-color: var(--colors-background-button) !important;
+}
+
+.btn-custom-primary:hover {
+  border-color: var(--colors-button-hover) !important;
```

Comment:
Can we refactor the orange button into a component so we don't need to keep adding custom css for it? I think we use this in many places right? This way we can limit the areas in the code where we break with Cloudscape convention.

Reply:
Created CustomOrangeButton component under components folder